datamodules/MNIST.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional, Callable
+import pytorch_lightning as pl
+from torch.utils.data import DataLoader, Subset
+from torchvision import transforms
+from torchvision.datasets import MNIST
+from sklearn.model_selection import train_test_split
+
+__all__ = ["MnistDataModule"]
+
+
+class MnistDataModule(pl.LightningDataModule):
+    def __init__(
+        self,
+        root_dir: str,
+        train_transforms: Callable,
+        test_transforms: Callable,
```

Comment:
생성자에 val_transforms 도 추가해야겠네 ... 
```suggestion
        val_transforms: Callable,
        test_transforms: Callable,
        
```

Reply:
근데 보통 val_transforms이랑 test_transforms이랑 동일한 augmentation 쓰니까 하나만 둬도 괜찮을 것 같은데